.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,33 @@
+on:
+  pull_request:
+    branches:
+    - master
+name: master Pull Request
+jobs:
+  test:
+    name: Run tests
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: install Singularity
+      run: |
+       bash .github/scripts/install_singularity.sh
+    - name: Build pipeline Singularity containers
+      run: |
+       echo $(which singularity)
+       bash scripts/build_singularity_containers.sh
+    - name: install Conda
+      run: bash .github/scripts/install_conda.sh
+    - name: install Nextflow via Conda
+      run: |
+       export PATH=/opt/conda/bin:$PATH
+       conda install -c bioconda nextflow
```

Comment:
since the test of the Conda cache requires Conda, I figured I could use it to install Nextflow as well with a one-liner. Do you prefer I install java 8/NF with a script?

Reply:
No, if it saves a manual install of java, the conda install is fine